acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/jsonrpc/Clique.java

```diff
@@ -50,6 +53,14 @@ public Clique(final EthTransactions eth, final CliqueTransactions clique) {
     this.clique = clique;
   }
 
+  public List<PantheonNode> validators(final PantheonNode[] nodes) {
```

Comment:
does the name represent what this function does? I.e. what does it do? Sort the params? Why is that a Clique thing?

Also - can probably be static

Reply:
not used anymore